index.html

```diff
@@ -120,12 +120,12 @@
         window.addEventListener(
             'error', loadErrHandler, true /* capture phase type of listener */);
     </script>
-    <script><!--#include virtual="/config.js" --></script><!-- adapt to your needs, i.e. set hosts and bosh path -->
```

Comment:
Do you have access to the jetty that you are using, can you add some handlers there? I cannot remember can this be added runtime or handlers are setup on startup of jetty...